bitnami/external-dns/templates/deployment.yaml

```diff
@@ -161,6 +161,7 @@ spec:
             {{- range .Values.sources }}
             - --source={{ . }}
             {{- end }}
+            {{- if eq .Values.provider "akamai" }}
```

Comment:
No closing end.
Missing akamai arguments.

Reply:
Looks like I had a merge error and the beginning "if" got separated from the end.  Fixed now.